src/components/Compare/CompareTable.tsx

```diff
@@ -155,6 +156,14 @@ const CompareTable = (props: {
   const isMobile = useMediaQuery(theme.breakpoints.down(700));
   const faqLinkCopy = isMobile ? 'FAQ' : 'Frequently Asked Questions';
 
+  // Disabling filters for Northern Mariana Islands because they don't have
+  // any data on metro vs non-metro islands.  There may be more elegant solutions
+  // that better handle any region without metro/non-metro regions.
+  const disableFilters = stateId === 'MP';
```

Comment:
So these 4 fips are all included in county_adjacency_msa.json--only 2 return a ‘nearby’ county, and a janky google search gives the impression that all 4 fall into the category of non-metro. Compare currently breaks+disappears when a filter selection returns nothing, so we can fix that (I prob should fix this regardless) and include the filters (though they’re not _super_ meaningful) if people are interested. Just an option !

Reply:
Yeah - maybe worth a trello ticket for handling this case better? I initially was hoping to disable the menu item based on whether or not the filters returned any locations, but that required a bit more work than this if statement haha